jenkins/job-configs/kubernetes-jenkins-pull/fejta-pull.yaml

```diff
@@ -0,0 +1,96 @@
+- job-template:
+    name: 'fejta-pull-{suffix}'
+
+    concurrent: true
+    properties:
+        - build-discarder:
+            days-to-keep: 7
+        - github:
+            url: 'https://github.com/{repo-name}'
+        - throttle:
+            max-total: '{max-total}'
+            max-per-node: 2
+            option: project
+        - raw:
+            xml: |
+                <com.cloudbees.plugins.JobPrerequisites plugin="slave-prerequisites@1.0">
+                    <script>docker version; gcloud version</script>
+                    <interpreter>shell script</interpreter>
+                </com.cloudbees.plugins.JobPrerequisites>
+    parameters:
+        - string:
+            name: sha1
+            description: 'sha1 or refname (e.g. origin/pr/N/head) to build'
+    triggers:
+        - github-pull-request:
+            # This is the Jenkins GHPRB plugin ID, not the actual github token.
+            auth-id: 'f8e31bc1-9abb-460a-a2ca-9c4aae3ca4e8'
+            only-trigger-phrase: true
+            trigger-phrase: '(?is).*@fejta do a little dance.*'
+            cron: 'H/2 * * * *'
+            status-context: Jenkins {status-context}
+            status-url: 'https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/${{ghprbPullId}}/${{JOB_NAME}}/${{BUILD_NUMBER}}/'
+            status-add-test-results: true
+            admin-list:
```

Comment:
can we at least add the kubernetes org as a whitelist? this is spamming *every* new PR twice.